src/js/themes/base.js

```diff
@@ -783,12 +783,18 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       xlarge: { ...fontSizing(2) },
       xxlarge: { ...fontSizing(4) },
     },
-    // textArea: {
-    //   extend: undefined,
-    // },
-    // textInput: {
-    //   extend: undefined,
-    // },
+    textArea: {
+      disabled: {
+        opacity: 0.3,
+      },
+      extend: undefined,
+    },
+    textInput: {
+      disabled: {
+        opacity: 0.3,
```

Comment:
where did the `0.3` come from? is it because we use this value for button disable? (which will make sense)

Reply:
yes it was from the button.